### ISSUE TITLE
Header hamburger menu is wrapping on mobile

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/component-library#40]
Header hamburger menu should be aligned to the right of the Stanford Libraries logo. 

<img src="https://github.com/user-attachments/assets/16e5d367-0e6a-4859-8fd0-db1af9a71c32" width="200">

**Comment by jcoyne** [comment]
I'm only seeing that problem on Earthworks, not other sites that use the component library:


<img width="523" alt="Screenshot 2024-08-28 at 10 33 09 PM" src="https://github.com/user-attachments/assets/b9cc011c-6316-4983-b2fb-1e73236ced90">


<img width="448" alt="Screenshot 2024-08-28 at 10 32 32 PM" src="https://github.com/user-attachments/assets/896ef9c0-16bd-42cd-95ca-b8f404f5be21">


I think it's due to this padding that was added around all containers there: https://github.com/sul-dlss/earthworks/blob/bl8/app/assets/stylesheets/earthworks.css#L9

**Comment by jcoyne** [comment]
@dbranchini to be clear, which devices to you call "on mobile".  I've tested mine on iPhone 11 and Galaxy S10 resolutions.

**Comment by dnoneill** [comment]
Yeah I had darcy put this in earthworks originally because I thought the same. Yesterday I did notice it is a problem in the component library but maybe the screensize is small enough that it doesn't matter.

<img width="523" alt="Screenshot 2024-08-29 at 11 02 44 AM" src="https://github.com/user-attachments/assets/665417ad-1114-474b-b400-dfd6a45995b1">
